.test-infra/kubernetes/influxdb/influxdb.yml

```diff
@@ -18,7 +18,7 @@ kind: Secret
 metadata:
   name: influxdb-creds
 data:
-  INFLUXDB_USER: c3VwZXJzYWRtaW4=
+  INFLUXDB_USER: c3VwZXJhZG1pbg==
```

Comment:
@bipinupd Could you elaborate from where these username/password came in? Where they are stored?

Reply:
The username/password are base64 encoded values. It can be anything, InfluxDB container creates the container based on that (env variables below). It is referenced when running IT test, 
https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_PerformanceTests_InfluxDBIO_IT.groovy#L38

and also used as default values in Pipeline Options in Java Code (https://github.com/apache/beam/blob/master/sdks/java/io/influxdb/src/test/java/org/apache/beam/sdk/io/influxdb/InfluxDBPipelineOptions.java#L33) .